### ISSUE TITLE
Provide reasons about the importance of the metrics

### ISSUE THREAD

**Issue by goern** [aicoe-aiops/ocp-ci-analysis#135]
As a consumer of the Black Flake metrics,
I want to get an idea of the importance of the metrics,
so that I can decide if I want to review the change of the metric over time

**Acceptance Criteria**

- [ ] ?

**Comment by MichaelClifford** [comment]
**TLDR** All current metrics are deemed important to some degree as they were sourced from research on current best practices. Beyond what we are currently doing (generating metrics to share with users, and documenting their value in KPI Request issues where needed) what else would you like to see to understand metric importance?   


@goern we have tried to track the meaning and content sources (blog, company website, ect) for each of the metrics in our working [spreadsheet](https://docs.google.com/spreadsheets/d/119WYsJU8ToTYOYP11lc6j_PdPefHXcfP5DCJjYzjrxQ/edit?usp=sharing), and you have been kind enough to add a "does it help" column that also provided some good insight into the relative importance of each metric. :smile: 

We have also aimed to provide more detailed reasons for why we want to track a specific metric in the KPI request issues (see #104 as an example) and I think moving forward having the reasons for collecting a specific metric be documented this way will be a valuable addition to this work. 

That said, for this initial phase, we are assuming that all of the metrics we are gathering are important to _someone_ in some degree as they were all sourced from researching AIOps and CI best practices. Furthermore, different metrics can be valuable to different people for different reasons, and as we move beyond single metrics and into some ML work, there maybe valuable combinations of metrics that aren't immediately obvious from the outset. So I don't necessarily want to limit what we are collecting at the moment.    

As we start to publish and share these metrics in the next few weeks, I am hoping we can get feedback from users as to which metrics are useful/ important, why so and what we are missing. Until then, I'm not sure what else we can do to provide information about individual metric importance. Let me know if you have any ideas :smile:
